src/utils/object.ts

```diff
@@ -32,7 +32,7 @@ export const transformKeys = (
         return transformArray(obj);
     }
 
-    const ignoredKeySet = new Set(ignoreKeys || []);
+    const ignoredKeySet = new Set(ignoreKeys);
```

Comment:
This is already handled in the default assignment. `null` is not a valid value here.